MaxText/tests/attention_test.py

```diff
@@ -40,6 +40,60 @@
 MLA = attentions.MLA
 
 
+class BidirectionalBlockMaskTest(unittest.TestCase):
```

Comment:
could you also test when B > 1, e.g. B == 2 with each batch element has different image position;

[[0, 1, 1, 1, 0, 0],
 [1, 1, 0, 0, 0, 0]]

Reply:
Good suggestion, I added test_batch_block_masks